debian/clearwater-cluster-manager.init.d

```diff
@@ -111,10 +111,10 @@ do_start()
                --log-directory=$log_directory
                --pidfile=$PIDFILE"
 
-  start-stop-daemon --start --quiet --pidfile $PIDFILE --exec $DAEMON --test > /dev/null \
+  start-stop-daemon --start --quiet --pidfile $PIDFILE --exec $ACTUAL_EXEC --test > /dev/null \
```

Comment:
Doesn't this change the executable to by "python"? How does this work? Why is it needed?